src/modules/auth/components/SetPasswordForm/SetPasswordForm.js

```diff
@@ -39,8 +40,17 @@ function SetPasswordForm({ onSubmit, recoveryPhrase }) {
     !password?.length || !cPassword?.length || !hasAgreed,
   [formValues.password, formValues.cPassword, formValues.hasAgreed]);
   const onFormSubmit = (values) => {
-    const accountSchema = encryptionAccount(recoveryPhrase, values.password);
-    onSubmit?.(accountSchema);
+    const accountSchema = encryptAccount({
+      recoveryPhrase: recoveryPhrase.value,
+      password: values.password,
+      name: values.accountName,
+    });
+
+    if (accountSchema.error) {
+      // we need to set error on a toast
```

Comment:
Is there a ticket for this? Will be good to add a in the todo "TODO #noftheticket"

Reply:
@isalga Resolved